### PR TITLE
add uv 0.1.9

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Notes:
- packages _just_ the `(bin|Scripts)/uv(.exe)` executable, not the python wrapper
  - a follow-on could add such as wrapper, but as there is no useful API at this point (just `run` which shells out), it wasn't worth the extra `python={x,y,z}` build dimension

### CODE REVIEW

**Comment by xhochy** [comment]
I think it might be better suited to build this directly from the GitHub source. Then we might be able to avoid including Python at all.

**Comment by bollwyvl** [comment]
> from the GitHub source

Sure, can try that to get down to just the three builds of the `(bin|Scripts)/uv(.exe)`.

**Comment by xhochy** [comment]
Happy to help if there are any problems but for me the current setup via PyPI seems way too complex to sustainably maintain.

File: recipes/uv/bld.bat [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    --locked ^
+    --path . ^
+    --profile release ^
+    --root "%PREFIX%" ^
+    || exit 1
+
+md "%PREFIX%\Scripts" ^
+    || echo "`%PREFIX%\Scripts` already exists"
+
+move "%PREFIX%\bin\uv.exe" "%PREFIX%\Scripts\uv.exe" ^
```
Comment (xhochy) [inline recipes/uv/bld.bat:15]: What's the reason you put this here? Normally, we would place it in `%LIBRARY_PREFIX%\bin`

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by bollwyvl** [comment]
Sure, can update for that path.

File: recipes/uv/bld.bat [hunk 2]
```diff
@@ -0,0 +1,24 @@
+@echo on
+
+cd crates\uv
+
+cargo install ^
+    --locked ^
+    --path . ^
+    --profile release ^
+    --root "%PREFIX%" ^
```
Comment (xhochy) [inline recipes/uv/bld.bat:9]:
```suggestion
    --root "%LIBRARY_PREFIX%" ^
```

File: recipes/uv/bld.bat [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    || exit 1
+
+md "%LIBRARY_PREFIX%\bin" ^
+    || echo "`%LIBRARY_PREFIX%\bin` already exists"
+
+move "%PREFIX%\bin\uv.exe" "%LIBRARY_PREFIX%\bin\uv.exe" ^
+    || exit 2
```
Comment (xhochy) [inline recipes/uv/bld.bat:16]:
```suggestion
```

**Review by xhochy — APPROVED** [review]

**Pull request merged**